libcxx/test/std/algorithms/alg.nonmodifying/alg.contains/ranges.contains.pass.cpp

```diff
@@ -0,0 +1,252 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Can you please go through various `robust` test files and add tests for `contains` where it makes sense? Let me know if you need any help with that!

Reply:
I add `robust` tests about projector, invoking and proxy iterators like `ranges::ends_with()` and `ranges::find()` but I don't add any `robust` tests about comparator because there is no comparator in `ranges::contains()`. Hope I miss nothing!